solutions/swb-reference/integration-tests/config/example.yaml

```diff
@@ -2,15 +2,22 @@
 ## Please deploy the swb-reference package before running the integration test suite.
 ## Settings declared here are merged with the main accounts CFN outputs to create integration test config items
 
-# Default Env Config for launching a new environment
+## Default Env Config for launching a new environment
 #envTypeId: ''
 #envTypeConfigId: ''
 #projectId: ''
 #envType: ''
 
-# Negative Testing
+## Negative Testing
 #terminatedEnvId: ''   # id of an environment that has already been terminated
 
-#Cognito Integ Test Client
+## Cognito Integ Test Client
 #rootUsername: ''
 #rootPasswordParamStorePath: ''
+
+## Pre-requisite for running aws-accounts integration test
+## Create the onboard_account stack in an external AWS account, and populate the following values below:
+# awsAccountId: ''
```

Comment:
Nitpick: Delete the extra space between `#` and `a`. Same for the lines 21 to 23